config/config.go

```diff
@@ -76,8 +76,8 @@ type PoliciesConfig struct {
 	// Set this to the URL of your Tyk Dashboard installation. The URL needs to be formatted as: http://dashboard_host:port.
 	PolicyConnectionString string `json:"policy_connection_string"`
 
-	// This option is required if `policies.policy_source` is set to `file`.
-	// Specifies the path of your JSON file containing the available policies.
+	// If `policies.policy_source` is set to `file` and `policies.policy_path` is not set. Then this option is required and will
```

Comment:
TBH I'm not sure both commends were clearer to me before, but will let @dcs3spp review it.

Reply:
Agreed seems clearer before. Is the following understanding correct?

- `PolicyRecordName` is inspected when `policies.policy_source`=`file` and `policies.policy_path` is not set. Tyk Gateway will read the JSON file in `policies.policy_path` containing all policies.
- `PolicyPath` is inspected when `policies.policy_source`=`file` and `policies.policy_path` is set. In this case the path specifies a directory containing JSON files. Does it try to read all *.json files?

Reply:
@letzya @dcs3spp we had a user ask us to improve the docs as they are not clear. Here is the [GH issue](https://github.com/TykTechnologies/tyk/issues/5485). I just took a stab at it but please feel free to recommend a better description of the options. 

@dcs3spp you are correct in your understanding. The only thing that I would add is if `policies.policy_source` is set to `file` or not set.

Please see the relevant code [here](https://github.com/TykTechnologies/tyk/blob/48abfdff71e792b907f7ffaa0b1a65de01d75c18/gateway/server.go#L558).